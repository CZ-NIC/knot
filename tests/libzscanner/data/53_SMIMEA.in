$ORIGIN	.
$TTL	1

; SMIMEA is the same as TLSA, so there are differences and basics only.

; OK
@	SMIMEA	0	0	0	00		; The simplest variant
@	TYPE53	\# 4 00000000				; TYPE + Hexadecimal rdata
@	TYPE53	0       0       0       00		; TYPE
@	smimea	0       0       0       00		; Type in lower-case

; KO
@	SMIMEA
