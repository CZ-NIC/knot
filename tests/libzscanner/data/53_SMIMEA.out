OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0035
RDATA=00000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0035
RDATA=00000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0035
RDATA=00000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0035
RDATA=00000000
------
WARNG=ZS_BAD_NUMBER
------
